docs/framework/react/community/community-projects.md

```diff
@@ -109,6 +109,12 @@ Generate React Query hooks based on Swagger API definitions
 
 Link: https://github.com/Shaddix/react-query-swagger
 
+## Suspensive React Query
+
+Enhances React Query with Suspense support, allowing for simpler and more declarative data fetching
+
+Link: https://github.com/toss/suspensive
```

Comment:
@gwansikk How about this?

```suggestion
Link: https://suspensive.org/docs/react-query/motivation
```

Reply:
Great! That document will provide a better explanation.